cdap-ui/app/cdap/components/Experiments/store/createExperimentStore.js

```diff
@@ -310,6 +310,23 @@ const active_step = (state = DEFAULT_ACTIVE_STEP, action = defaultAction) => {
     return state.active_step;
   };
   switch (action.type) {
+    case ACTIONS.SET_WORKSPACE_ID: {
+      const newState = {
```

Comment:
this doesn't throw an error? I thought you can't declare a new variable inside a switch statement

Reply:
I think what you are referring is declaring a variable a variable inside the case statement eventually gets hoisted to the top. So you can have two case statement declaring the same variable,

```
switch(...) {
  case 'One':
    let var1 = "var1value";
    ...
  case 'Two': 
   let var1 = "someothervalue";
   ...
}
```
this will error out. However in this case I am wrapping the code within a `{}` so it creates a scope block which is fine since the variable won't exist beyond the scope.